py3-certifi.yaml

```diff
@@ -36,9 +49,13 @@ pipeline:
   - runs: |
       python3 setup.py install --prefix=/usr --root="${{targets.destdir}}"
 
+      sed -i 's/Version: ${{vars.normalized-package-version}}/Version: ${{package.version}}/' "${{targets.destdir}}"/usr/lib/python3.11/site-packages/certifi-${{vars.normalized-package-version}}-py3.11.egg-info/PKG-INFO
```

Comment:
Would this need to change under py3.12+ ?

Reply:
I don't have any ideas, especially not any good ones.

Thinking it through, when 3.12 is released we won't automatically rebuild this package to use it, so we won't notice until its epoch gets bumped later, which will fail pretty mysteriously.

Can you add a comment like `TODO: Need to update this path when building using python-3.12`? That'd be enough of a breadcrumb I think.